nipoppy/env.py

```diff
@@ -9,7 +9,7 @@
 # BIDS
 BIDS_SUBJECT_PREFIX = "sub-"
 BIDS_SESSION_PREFIX = "ses-"
-FAKE_SESSION = "ses-IsFake"
+FAKE_SESSION_ID = "IsFake"
```

Comment:
```suggestion
FAKE_SESSION_ID = "unnamed"
```

I checked with the Neurobagel team and @nikhil153 and they prefer `unnamed` over `IsFake`